tests/FixturesTraitTest.php

```diff
@@ -73,6 +73,23 @@ public function testExportJson()
         $this->assertFileExists($this->getFixturePath('export_json/response.json'));
     }
 
+    public function testFixtureWithoutExtension()
+    {
+        putenv('FAIL_EXPORT_JSON=false');
+
+        $result = [
+            'value' => 1234567890,
+        ];
+
+        $this->exportJson('export_json/response', new TestResponse(
+            new Response(json_encode($result))
+        ));
+
+        $this->assertEquals($this->getJsonFixture('export_json/response'), $result);
+
+        $this->assertFileExists($this->getFixturePath('export_json/response.json'));
```

Comment:
please remove file in start of test to be sure that artefacts from the previous run is not affect for the current one